haas/ext/auth/database.py

```diff
@@ -0,0 +1,149 @@
+"""Auth plugin using usernames & passwords in the DB, with HTTP basic auth.
+
+Includes API calls for managing users.
+"""
+from haas import api, model, auth
+from haas.auth import get_auth_backend
+from haas.rest import rest_call, local
+from haas.errors import *
+from sqlalchemy import Column, ForeignKey, String, Boolean, Table
+from sqlalchemy.orm import relationship
+from passlib.hash import sha512_crypt
+from schema import Schema, Optional
+
+
+class User(model.Model):
+    """A user of the HaaS.
+
+    A user can be a member of any number of projects, which grants them access
+    to that process's resources. A user may also be flagged as an administrator.
+    """
+
+    is_admin = Column(Boolean, nullable=False)
```

Comment:
Sure. Keep in mind you'll want to make sure to do the corresponding SQL thing when doing the migration; the one-liner I gave above doesn't do this.